### ISSUE TITLE
[Request] New App Request

### ISSUE THREAD

**Issue by animetrom** [3dproger/AxelChat#447]
Would it be possible to develop a very simplified app for Android that points every chat open on Windows (over the network) to this application?
This request is due to not having many tools to help you stream through Meta Quest (VR).
Yes, I understand that there are several solutions with PCVR, however, I wanted an alternative without complete dependence on a computer.
If this is not possible, if you can create a solution that allows display locally (localhost), it would help me a lot.

**Comment by 3dproger** [comment]
Yes it is possible. Moreover, the application on Android has already been compiled and launched, but it doesn’t work very well yet and I don’t want to release it in such a raw form. A mobile release is planned eventually, but it's hard to say when that will happen.

**Comment by animetrom** [comment]
Thanks

**Comment by 3dproger** [comment]
There is no need to close this issue. An open ticket will be an additional reminder for me issue it is completed

**Comment by animetrom** [comment]
@3dproger Please, would you know a way for me to get the url of the widget that we placed in obs studio to show the chat, leaving (remotely locally) a way to open it in the browser, but, for example, on my cell phone?

**Comment by 3dproger** [comment]
This cannot be done yet, but I will try to make it possible. Added a issue to implement this: https://github.com/3dproger/AxelChat/issues/457
